### ISSUE TITLE
Can we transform the blog into a `planet`?

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy-hub#420]
The idea here is that the blog also lists posts from other servers like https://usegalaxy-eu.github.io/news and be an aggregator ... http://planetpython.org/ or http://planet.gnome.org/ are such examples.

This does not mean we can not post here directly, but adding posts from other might be nice. This could be also done via RSS/Atom maybe?

**Comment by dannon** [comment]
Our first pass of this was to have the blog links like this example: https://github.com/galaxyproject/galaxy-hub/blob/master/src/blog/2017-12-galaxy-on-digitalocean/index.md, which show up in all the expected feeds but actually keep all the relevant content externally -- you just have to create the metadata 'pointer'.

If I write a script that'll just autogenerate these pointers, consuming the feed at https://usegalaxy-eu.github.io/feed.xml (and other desired locations), would that work for you?

**Comment by bgruening** [comment]
> If I write a script that'll just autogenerate these pointers, consuming the feed at https://usegalaxy-eu.github.io/feed.xml (and other desired locations), would that work for you?

Sure, thanks!

**Comment by dannon** [comment]
Cool, that should be pretty easy to hack out.  Do you know of other sources off the top of your head, other than the usegalaxy-eu news feed?

**Comment by bgruening** [comment]
No, sorry. Also the news feed from EU maybe needs to be filtered according to tags, maybe ... lets see how it looks and we can enhance this later.

Thanks!

**Comment by tnabtaf** [comment]
Heh, I was thinking of @pjbriggs [Galactic Engineer Blog](http://galacticengineer.blogspot.com/).  For instance I just missed a post he made.  I'll add it now, but it would be nice to pick those up automatically in the future.  No filtering required.
